server.js

```diff
@@ -12,37 +15,33 @@ app.get('/', (request, response) => {
   response.send('TasteRX API');
 });
 
-app.listen(app.get('port'), () => {
-  console.log(`${app.locals.title} is running on http://localhost:${app.get('port')}.`);
-});
-
-app.locals.counter = 1;
-app.locals.prescriptions = [];
-
-app.post("/prescriptions", (request, response) => {
+app.post("/prescriptions", async (request, response) => {
   const { message, showID } = request.body
   if(!message || !showID) {
-    response.status(422).json( { error: "Expected { message: <String>, showID: <Number>}" } )
-  }
-
-  const newPrescription = {
-    id: app.locals.counter,
-    message: message,
-    showID: showID
+    response.status(422).json( { error: "Expected { message: <String>, showID: <Number> }" } )
   }
-  app.locals.counter ++
-
-  app.locals.prescriptions.push(newPrescription)
-  response.status(201).json({ prescription: newPrescription})
+  const newPrescription = await models.TVPrescription.create({ message: message, showID: showID })
+  response.status(201).json({ prescription: newPrescription })
 })
 
```

Comment:
@nicktu12 these are async because `create` and `findOne` (the query operations) return a promise.